fabric-structure-api-v1/src/main/java/net/fabricmc/fabric/mixin/structure/StructurePoolAccessor.java

```diff
@@ -31,6 +32,7 @@
 	List<StructurePoolElement> getElements();
 
 	@Accessor(value = "elementCounts")
+	@Mutable
```

Comment:
Should be on the setter I think, but mixin might apply the `final` removal here too (even though it's a bit weird)

Reply:
Okay, I'll try to see what happens when I remove it.

Reply:
Seems to work fine, now it is just on the setter.